### PR TITLE
Add support for flow type spread in 0.43

### PR DESCRIPTION
**Pull request by vjeux**
Fixes https://twitter.com/ryyppy/status/844116626467438592

### CODE REVIEW

**Comment by vjeux** [comment]
Here's the (not merged yet) PR on babylon https://github.com/babel/babylon/pull/418

**Comment by vjeux** [comment]
Now that we are on babylon beta8 (thanks @existentialism), it also works for babylon!

File: tests/flow_type_spread/jsfmt.spec.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+run_spec(__dirname);
+run_spec(__dirname, {parser: 'flow'});
```
Comment (existentialism) [inline tests/flow_type_spread/jsfmt.spec.js]: Should be `parser: "babylon"` since default is flow, yeah?

Comment (vjeux) [inline tests/flow_type_spread/jsfmt.spec.js]: Woops, good call! I'll fix this :) I tested it on the command line and it's working on both.

**Pull request merged**